flow/platforms/asap7/config.mk

```diff
@@ -99,18 +96,20 @@ export PLACE_DENSITY ?= 0.60
 export TAPCELL_TCL             = $(PLATFORM_DIR)/openRoad/tapcell.tcl
 
 # TritonCTS options
-export CTS_BUF_CELL            ?= BUFx4_ASAP7_75t_R
+export CTS_BUF_CELL            = BUFx4_ASAP7_75t_R
```

Comment:
Why remove the ?

Reply:
I wanted to be able to change the buffer cell via my design config. I can also remove it but this would make that possible.

Reply:
Ah, wait yeah, you are right let me add that again :-)